litho-widget/src/main/java/com/facebook/litho/widget/RenderInfoViewCreatorController.java

```diff
@@ -46,7 +46,7 @@ public RenderInfoViewCreatorController(boolean customViewTypeEnabled, int compon
 
   @UiThread
   public void maybeTrackViewCreator(RenderInfo renderInfo) {
-    if (!renderInfo.rendersView()) {
+    if (!renderInfo.rendersView() || renderInfo == AdapterProxy.PROXY_RENDER_INFO) {
```

Comment:
This seems weird! If `AdapterProxy` is going to be used, maybe then `RenderInfo` should have a method that will indicate whether it should be tracked, otherwise it risks being unmaintainable in case more `RenderInfo` children will be added in the future!